### ISSUE TITLE
Glenohumeral (shoulder) arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11005]
**Preferred term label:**
Glenohumeral (shoulder) arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
AP, axillary, or Grashey (true AP) view x-ray of the shoulder showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the glenohumeral joint

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Rotator cuff tear arthropathy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001458
- label: Glenohumeral arthritis
- parents: [Abnormal shoulder morphology (HP:0003043)](https://hpo.jax.org/app/browse/term/HP:0003043)
- pmid: PMID:32662351
- def: Anteropostrior (AP), axillary, or Grashey (true AP) view X-ray of the shoulder showing radiographic evidence of osteoarthritis such as joint space narrowing, subchondral sclerosis, subchondral cysts, and/or osteophyte formation in the glenohumeral joint.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11005
- synonyms: Shoulder arthritis(NONE)
